iroh/tests/cli.rs

```diff
@@ -612,22 +601,28 @@ where
                 break;
             }
 
-            if let Some(Ok(line)) = lines.peek() {
-                println!("|{}", line);
+            match lines.peek() {
+                Some(Ok(line)) => {
+                    println!("|{}", line);
 
-                if let Some(cap) = rx.captures(line) {
-                    for i in 0..cap.len() {
-                        if let Some(capture_group) = cap.get(i) {
-                            caps.push(capture_group.as_str().to_string());
+                    if let Some(cap) = rx.captures(line) {
+                        for i in 0..cap.len() {
+                            if let Some(capture_group) = cap.get(i) {
+                                caps.push(capture_group.as_str().to_string());
+                            }
                         }
-                    }
 
-                    matches += 1;
-                } else {
-                    break;
+                        matches += 1;
+                    } else {
+                        break;
+                    }
+                }
+                Some(Err(err)) => {
+                    panic!("Error from reader: {err:#}");
+                }
+                None => {
+                    panic!("All lines read but no match found for /{rx}/");
```

Comment:
Any chance we can print all output here, for easier debugging?

Reply:
the output read so far is already printed above, and getting here means there is no more output afaik so nothing more to print.

Reply:
ah okay, great